.env.example

```
@@ -16,7 +17,7 @@ R2_PUBLIC_DOMAIN=YOUR-R2-PUBLIC-DOMAIN
 R2_SECRET_ACCESS_KEY=YOUR-R2-SECRET-ACCESS-KEY
 REDIS_URL="redis://127.0.0.1:6379"
 TURNSTILE_SECRET_KEY=YOUR-TURNSTILE-SECRET-KEY
-UPLOAD_DIR=uploads // optional if the user upload saved in localstorage, default is "uploads"
+UPLOAD_DIR=uploads // optional if the user upload saved in localstorage or you dont have any cf storage bucket (you might have to change the upload logic code to the localstorage e.g attachment, user/group avatar, and story), default is "uploads"
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)